### PR TITLE
[Wallet] Fix contact thumbnails in tx feed and related cleanup

### LINKED ISSUE #837: Contact resolution doesn't work for txs in activity feed
### Expected Behavior

Transactions on home screen show correctly

### Current Behavior

Avatars (contact pictures) show up correctly in send screen, but not on home page after the transaction sends.

### PR DESCRIPTION
**Pull request by jmrossy**
### Description

- Fix contact thumbnails in tx feed 
- Remove duplicated logic around avatars
- Remove now obsolete AccountInfo component
- Add simple component for the user's own avatar
- Fix Transaction type enum to match value in blockchain api (not sure why this worked as is, must be graphql magic)
- Related cleanup

### Tested

Loaded affected screens, all LGTM, including thumbnail in feed (see screenshots)

### Related issues

- Fixes #837 

### Backwards compatibility

Yes

wa-feed-picture
wa-qr-code

### CODE REVIEW

File: packages/mobile/src/account/reducer.ts [hunk 1]
```diff
@@ -64,11 +64,11 @@ export const reducer = (state: State | undefined = initialState, action: ActionT
         defaultCountryCode: action.countryCode,
       }
     case Actions.DEV_MODE_TRIGGER_CLICKED:
-      const newClickCount = (state.devModeClickCount + 1) % 10
+      const newClickCount = (state.devModeClickCount + 1) % 6
```
Comment (jeanregisser) [inline packages/mobile/src/account/reducer.ts:67]: nit: this seems a bit overkill to store the number of clicks in the redux store. Just a remark and unrelated to your changes of course ;)

File: packages/mobile/src/components/Avatar.tsx [hunk 2]
```diff
@@ -8,81 +8,59 @@ import { Avatar as BaseAvatar } from '@celo/react-components/components/Avatar'
...
+  throw new Error('Invalid avatar props, cannot determine display name')
 }
 
-type AvatarProps = Props & StateProps & WithNamespaces
-
-export class Avatar extends React.PureComponent<AvatarProps> {
-  render() {
-    const { t, recipient, address, e164Number, iconSize = DEFAULT_ICON_SIZE } = this.props
+export function Avatar(props: Props) {
+  const defaultCountryCode = useSelector(defaultCountryCodeSelector)
```
Comment (jeanregisser) [inline packages/mobile/src/components/Avatar.tsx:46]: 👍

**Review by jeanregisser — APPROVED** [review]
Very nice! 👍

File: packages/mobile/src/account/reducer.ts (hunk 1 above)
Comment (jmrossy) [reply packages/mobile/src/account/reducer.ts:67]: Yeah, I guess it could be a module-level variable instead.

**Pull request merged**